index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portofolio</title>
+</head>
+<body>
+    <header class="header">
+      <div class="logo">My Logo</div>
+     
```

Comment:
- [ ] For semantic reason it could be better if you use `a` tag for the logo instead of a `div` 😄